### ISSUE TITLE
Alligators on Mississippi

### ISSUE THREAD

**Issue by coastypoo** [kk49/deca_website#39]
they aint non

**Comment by barryb777** [comment]
Yep, alligators are missing on mississippi +1 (I think they are being read under Green winged teals)

**Comment by DNLzz99** [comment]
> Yep, alligators are missing on mississippi +1 (I think they are being read under Green winged teals)

They are still missing. I don't think they are listed under Green Winged Teal, because those were added to the reserve with the latest update.

**Comment by kk49** [comment]
Please go to #60
